server/base/src/main/java/org/apache/accumulo/server/conf/TableConfiguration.java

```diff
@@ -144,7 +146,7 @@ public ParsedIteratorConfig getParsedIteratorConfig(IteratorScope scope) {
   private static ScanDispatcher createScanDispatcher(AccumuloConfiguration conf,
       ServerContext context, TableId tableId) {
     ScanDispatcher newDispatcher = Property.createTableInstanceFromPropertyName(conf,
-        Property.TABLE_SCAN_DISPATCHER, ScanDispatcher.class, null);
+        Property.TABLE_SCAN_DISPATCHER, ScanDispatcher.class, new SimpleScanDispatcher());
```

Comment:
There's a nice logging block in the compactionDispatcher code that logs an error if null. 
Not sure if that also exists in the `createScanDispatcher` method. 

The only thing we want to avoid is somehow returning the default class due to a transient classloader issue and then having the same problem where the default class is referenced by the Deriver and it cannot be updated without changing the property value to a non-standard value to trigger the class reload.